README.md

```diff
@@ -1,24 +1,217 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* Ruby version
+<!-- PROJECT DESCRIPTION -->
+# 📖 MyBudget <a name="about-project"></a>
 
-* System dependencies
+> MyBudget application 
 
-* Configuration
+** Mybudget ** is a ruby project where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+<!-- 
+<div style="position: relative; padding-bottom: 56.25%; height: 0;"><iframe src="https://www.loom.com/embed/18895e5ca97a4224b5b5add9f127a99c" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div> -->
 
-* Database creation
+[![Watch the video](./project.png)](https://www.loom.com/embed/18895e5ca97a4224b5b5add9f127a99c)
 
```

Comment:
- [OPTIONAL] It will be nice to make the link to your video presentation more explicit, it is difficult for one to know that the image is the link to the video presentation. Please kindly fix that.

Reply:
Would work on it.